[2023-11-06T12:08:09.304+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-11-02T00:00:00+00:00 [queued]>
[2023-11-06T12:08:09.320+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-11-02T00:00:00+00:00 [queued]>
[2023-11-06T12:08:09.321+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-06T12:08:09.349+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): task_a> on 2023-11-02 00:00:00+00:00
[2023-11-06T12:08:09.356+0000] {standard_task_runner.py:57} INFO - Started process 1051 to run task
[2023-11-06T12:08:09.363+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'catchup_and_backfill_dag', 'task_a', 'scheduled__2023-11-02T00:00:00+00:00', '--job-id', '123', '--raw', '--subdir', 'DAGS_FOLDER/catchup_and_backfill_dag.py', '--cfg-path', '/tmp/tmpwzdm0ziw']
[2023-11-06T12:08:09.369+0000] {standard_task_runner.py:85} INFO - Job 123: Subtask task_a
[2023-11-06T12:08:09.492+0000] {task_command.py:416} INFO - Running <TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-11-02T00:00:00+00:00 [running]> on host cf1baac588aa
[2023-11-06T12:08:09.654+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='safarnejad' AIRFLOW_CTX_DAG_ID='catchup_and_backfill_dag' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2023-11-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-02T00:00:00+00:00'
[2023-11-06T12:08:09.657+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-06T12:08:09.658+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello World :)']
[2023-11-06T12:08:09.672+0000] {subprocess.py:86} INFO - Output:
[2023-11-06T12:08:09.676+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: syntax error near unexpected token `)'
[2023-11-06T12:08:09.677+0000] {subprocess.py:93} INFO - /bin/bash: -c: line 1: `echo Hello World :)'
[2023-11-06T12:08:09.678+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-11-06T12:08:09.696+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-11-06T12:08:09.703+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=catchup_and_backfill_dag, task_id=task_a, execution_date=20231102T000000, start_date=20231106T120809, end_date=20231106T120809
[2023-11-06T12:08:09.725+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 123 for task task_a (Bash command failed. The command returned a non-zero exit code 2.; 1051)
[2023-11-06T12:08:09.739+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-06T12:08:09.771+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-06T12:19:12.939+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-11-02T00:00:00+00:00 [queued]>
[2023-11-06T12:19:12.959+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-11-02T00:00:00+00:00 [queued]>
[2023-11-06T12:19:12.962+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-11-06T12:19:13.023+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): task_a> on 2023-11-02 00:00:00+00:00
[2023-11-06T12:19:13.031+0000] {standard_task_runner.py:57} INFO - Started process 2103 to run task
[2023-11-06T12:19:13.039+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'catchup_and_backfill_dag', 'task_a', 'scheduled__2023-11-02T00:00:00+00:00', '--job-id', '179', '--raw', '--subdir', 'DAGS_FOLDER/catchup_and_backfill_dag.py', '--cfg-path', '/tmp/tmpe0116pon']
[2023-11-06T12:19:13.046+0000] {standard_task_runner.py:85} INFO - Job 179: Subtask task_a
[2023-11-06T12:19:13.188+0000] {task_command.py:416} INFO - Running <TaskInstance: catchup_and_backfill_dag.task_a scheduled__2023-11-02T00:00:00+00:00 [running]> on host cf1baac588aa
[2023-11-06T12:19:13.385+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='safarnejad' AIRFLOW_CTX_DAG_ID='catchup_and_backfill_dag' AIRFLOW_CTX_TASK_ID='task_a' AIRFLOW_CTX_EXECUTION_DATE='2023-11-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-02T00:00:00+00:00'
[2023-11-06T12:19:13.388+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-06T12:19:13.389+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo Hello World.']
[2023-11-06T12:19:13.405+0000] {subprocess.py:86} INFO - Output:
[2023-11-06T12:19:13.407+0000] {subprocess.py:93} INFO - Hello World.
[2023-11-06T12:19:13.409+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-11-06T12:19:13.459+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=catchup_and_backfill_dag, task_id=task_a, execution_date=20231102T000000, start_date=20231106T121912, end_date=20231106T121913
[2023-11-06T12:19:13.504+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-06T12:19:13.536+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
